NAMESPACE

```diff
@@ -151,8 +151,11 @@ S3method("+", IDate)
 S3method("-", IDate)
 S3method(as.character, ITime)
 S3method(as.data.frame, ITime)
-S3method(as.Date, IDate)   # note that zoo::as.Date masks base::as.Date. Both generic.
-export(as.Date.IDate)      # workaround for zoo bug, see #1500. Removing this export causes CI pipeline to fail on others.Rraw test 6, but I can't reproduce locally.
+if (getRversion() >= "3.6.0") {
+  S3method(as.Date, IDate)
```

Comment:
@jangorecki Why >= "3.6.0"?  Won't `as.Date(x)` break in R < 3.6.0 if the method is not exported?
Also could the comments be put back so we know there's some history on these lines pointing to #1500.

Reply:
Higher up in the file the switch on 3.6.0 is about delayed registration (i.e. the `pkg::` prefix on the first argument to `S3method`) but that's not the case here with `as.Date`.

Reply:
I figured the >=3.6 was copied from higher up. So I just went ahead and removed it, to merge and let's see if it passes CI. If this was wrong then we can always do a follow up PR.

Reply:
When I added the suggest version for zoo, I did bit64 and bit at the same time (although nothing to do with this PR).